lib/google/cloud/gemserver/cli/server.rb

```diff
@@ -182,15 +183,36 @@ def sanitize_name name
           # @private Outputs helpful information to the console indicating the
           # URL the gemserver is running at and how to use the gemserver.
           def display_next_steps
-            puts "The gemserver has been deployed! It is running on #{remote}"
-            puts "To see how to use your gemserver to push and download gems" \
-              "read https://github.com/GoogleCloudPlatform/google-cloud-" \
+            puts "\nThe gemserver has been deployed! It is running on #{remote}"
+            puts "To see the status of the gemserver, visit: \n" \
+              " #{remote}/health"
+            puts "\nTo see how to use your gemserver to push and download " \
+              "gems read https://github.com/GoogleCloudPlatform/google-cloud-" \
               "gemserver/blob/master/docs/usage_example.md for some examples."
-            puts "For general information, visit https://github.com/" \
+            puts "\nFor general information, visit https://github.com/" \
               "GoogleCloudPlatform/google-cloud-gemserver/blob/master/README.md"
           end
 
           ##
+          # @private Pings the gemserver until a timeout or the gemserver
+          # replies with a 200 response code.
+          #
+          # @param [Integer] timeout The length of time the gemserver is
+          # pinged. Optional.
+          def wait_until_server_accessible timeout = 60
+            puts "Waiting for the gemserver to be accessible..."
+            start_time = Time.now
+            loop do
+              if Time.now - start_time > timeout
+                fail "Could not establish a connection to the gemserver"
+              else
+                r = Request.new(nil, @config[:proj_id]).health
```

Comment:
I'd put in a bit of sleep in between checks (a second should be adequate) just to throttle things a bit.